### PR TITLE
[zh-cn] Translation blog: legacy-package-repository-deprecation

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Fixes:  https://github.com/kubernetes/website/issues/42953

Here is [the preview page](https://deploy-preview-42955--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/).

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg
Comment (sftim) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg:1]: Extra credit: use Chinese writing (you can use a tool like Inkscape to replace the text with Chinese text, or you can make a new diagram using Mermaid).

[Mermaid source](https://mermaid.live/edit#pako:eNqtlVFv2jAQx7-K5z6slQKFlEDIJqaq3fbQdqtGp6orPJj4AhHGjmxnlAHffbYDKwlU7cN4CLmL_bu7_52TJY4FBRzhhIl5PCFSo7vLAUfmp7Sxjh_79m94gmq1HppPiL5KOf2eXIVqeW8sNDUmEsnHkTztXeUjkBw0KBSzXGmQxt2jAi1EjibkN3xaF-jiWqJZ_qoPLEEzwskY6Grn-XXK86dqvJ7IQBKd8jFSCxNs9kKwCsYFuoRRSnhtRBRQ5NwOKNGP25td7wpxAKrO-c-MEg3Hjw8GPiML50ZaoNz57eaMxFOT9yZ9iWLBk3Sc2wwFd3RuU5MFGHEjuxqeFCmWgqBavYYUwMUE4mnheuwDoInWmYpOT6ehqqfilFmMhEyo4aZfpS2uX7G1bVdizQb8ZTkuBNck5SAPKHJv1oq5WpVgy3MJTuhcWf2f129qvL0ot3p3r4v4AKqi7bWICfuPAt9eHFTXhTkgsf8mjffq-CZWaCwEvRNfhc39vdHF2jb1sXg33Ffdzrrb-iVnbLGddluNloKhjBHuyiVmzt2QrCptfAF3cwjkuXOpdo7VlpkJDVynhL0-2U7UfcZW3grJJFPrVYbMqmAWx4wodQmJzS3lKEkZi44opZ4yGU8hOkqSZHNfm6dUT6JW9uTFggkZHTUajQ-7DNOiDeHMb8cQvAlinv2DFC84rzQd3v6seOU5qZi-t-1-UdQzvdQiryKIVxHNKw2WKa1UKtAxXJMRMDQyZ2AsRc5pUWchwHPFr0uIPTwDOSMpNS_9pe3gAOsJzGCAI3NLISE50wM84GuztBiCzzTVQuIoIUyBh0muRX_BYxxpmcN20WVKxpLMts6McBwt8ROOmu2g7oeNoBO2m2d-2PQDDy9wVOt262EjbAetThg0g7YfrD38RwhDaNS7Qdf3W-FZt9PqdBpB1_F-uYcFHlxKN8W3y33C1n8BsW9d2w) (editable)

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+
+<!--
+The following diagram shows who's affected by this change in a visual form
+(click on diagram for the larger version):
+
+{{< figure src="/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" alt="Visual explanation of who's affected by the legacy repositories being deprecated and frozen. Textual explanation is available above this diagram." class="diagram-large" link="/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" >}}
+-->
+下图以可视化形式显示了谁受到此更改的影响（单击图表可查看大图）：
+
+{{< figure src="/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" alt="直观地解释谁受到弃用和冻结的遗留仓库的影响。图上提供了文字解释。" class="diagram-large" link="/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" >}}
```
Comment (sftim) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:127]:
```suggestion
{{< figure src="/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" alt="直观地解释谁受到弃用和冻结的遗留仓库的影响。图上提供了文字解释。" class="diagram-large" link="/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" >}}
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg
Comment (sftim) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg:1]: Maybe also change the URL to be a localized one.

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+This change affects users **directly installing upstream versions of Kubernetes**,
+either manually by following the official
+[installation](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) and
+[upgrade](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) instructions, or
+by **using a Kubernetes installer** that's using packages provided by the Kubernetes
+project.
+-->
+## 作为 Kubernetes 最终用户，这对我有何影响？ {#how-does-this-affect-me-as-a-kubernetes-end-user}
+
+此更改影响用户**直接安装 Kubernetes 的上游版本**，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:66]: 是不是这样理解更好？

```suggestion
此更改影响**直接安装 Kubernetes 的上游版本**的用户，
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+无论是按照官方手动[安装](/zh-cn/docs/setup/生产环境/工具/kubeadm/install-kubeadm/)
+和[升级](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)说明，
+还是通过**使用 Kubernetes 安装器**，该安装器使用 Kubernetes 项目提供的软件包。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:69]:
```suggestion
还是通过**使用 Kubernetes 安装工具**，该安装工具使用 Kubernetes 项目提供的软件包。
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+你才会受到此更改的影响。云提供商通常使用他们自己的 Kubernetes 发行版，
+因此他们不使用 Kubernetes 项目提供的软件包；更重要的是，如果有其他人为你管理 Kubernetes，
+那么他们通常会负责该检查。
+
+<!--
+If you have a managed [control plane](/docs/concepts/overview/components/#control-plane-components)
+but you are responsible for **managing the nodes yourself**, and any of those nodes run Linux,
+you should [check](#check-if-affected) whether you are affected.
+-->
+如果你有托管的[控制平面](/zh-cn/docs/concepts/overview/components/#control-plane-components)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:97]:
```suggestion
如果你使用的是托管的[控制平面](/zh-cn/docs/concepts/overview/components/#control-plane-components)
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+但你负责**管理自己的节点**，并且每个节点都运行 Linux，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:98]:
```suggestion
但你负责**自行管理节点**，并且每个节点都运行 Linux，
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+  Kubernetes will freeze the legacy package repositories,
+  (`apt.kubernetes.io` and `yum.kubernetes.io`).
+  The freeze will happen immediately following the patch releases that are scheduled for September, 2023.
+-->
+## 变更时间表  {#timeline-of-changes}
+
+<!-- note to maintainers - the trailing whitespace is significant -->
+
+- **2023 年 8 月 15 日：**  
+    Kubernetes 宣布推出一个新的社区管理的 Kubernetes 组件 Linux 软件包源
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:159]: 不要缩进四个空格，下同。

```suggestion
  Kubernetes 宣布推出一个新的社区管理的 Kubernetes 组件 Linux 软件包源
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+**你将无法升级到自该日期起发布的任何补丁或次要版本。**
+
+<!--
+Whilst the project makes every effort to release secure software, there may one
+day be a high-severity vulnerability in Kubernetes, and consequently an important
+release to upgrade to. The advice we're announcing will help you be as prepared for
+any future security update, whether trivial or urgent.
+-->
+尽管该项目会尽一切努力发布安全软件，但有一天 Kubernetes 可能会出现一个高危性漏洞，
+因此需要升级到一个重要版本。我们宣布的建议将帮助你为未来的任何安全更新（无论是微不足道的还是紧急的）做好准备。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:232]:
```suggestion
因此需要升级到一个重要版本。我们所公开的建议将帮助你为未来的所有安全更新（无论是微不足道的还是紧急的）做好准备。
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+have provided their time; Google as a business has provided both the infrastructure
+to serve packages, and the security context for giving those packages trustworthy
+digital signatures.
+These have been important to the adoption and growth of Kubernetes.
+-->
+## 致谢  {#acknowledgments}
+
+首先，我们要感谢 Alphabet 的贡献。 Google 的员工投入了他们的时间；
+作为一家企业，谷歌既提供了服务于软件包的基础设施，也提供了为这些软件包提供可信数字签名的安全上下文。
+这些对于 Kubernetes 的采用和增长非常重要。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:396]:
```suggestion
这些对于 Kubernetes 的采用和成长非常重要。
```

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg:1]: Thank you very much. I did not find the Mermaid source file before.
I will open a new PR to follow up in a few days.

Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/flow.svg:1]: Thank you very much. I did not find the Mermaid source file before.
I will open a new PR to follow up in a few days.

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:127]: Updated.

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:66]: 也可以。

File: content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md (hunk 3 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md]: Updated.

**Comment by windsonsea** [comment]
/lgtm

The .svg can be localized later.

**Pull request merged**